ecosystem/opUSDC/op-usdc.md

```diff
@@ -0,0 +1,112 @@
+
+# Problem Statement + Context
+
+USDC is one of the most widely used assets across the crypto ecosystem, often utilized for bridging when new chains emerge before any action from Circle. This situation presents recurring challenges when bridged USDC gains significant market share, but native USDC is preferred by the ecosystem, leading to fragmentation between multiple representations of USDC.
+
+To address this pain point, Circle has rolled out the [Bridged USDC Standard](https://www.circle.com/blog/bridged-usdc-standard) to ensure that chains can easily migrate into the Native USDC version of the token in the future and prevent the fragmentation problem.
+
+However, the Bridged USDC Standard as defined does not work with the current OP Stack `StandardBridge` specifications, causing confusion from chain operators on how to conveniently host USDC on their chain. Circle’s primary goal is to enable chain developers to easily deploy the Bridged USDC Standard. 
+
+To address the issues faced by chain operators and align with Circle's motivations, OpUSDC is proposed as a reference implementation for OP Stack chain operators. This allows them to comply with the Bridged USDC standard and potentially access native USDC.
+
+# Alternatives Considered
+
+**Do nothing**
+
+The `StandardBridge` contracts have been implemented since the chain's inception, allowing the use of any common ERC20 token, such as USDC. As the number of OP Chains within the Superchain ecosystem grows, a problematic scenario will appear where an OP Chain might host different implementations of the same asset: *legacy* bridged USDC, new bridged USDC that accomplish with the Superc20, 3rd party bridged USDC, and native USDC. This fragmentation would significantly complicate the migration to native USDC and negatively impact the user experience.
+
+**Dedicated shared bridge**
+
+To align architecturally with the Superchain design, a proposal was made to enable full fungibility of opUSDC across OP Chains within the same interoperability cluster. This design featured a `SharedL1Adapter` and a factory that allowed trustless deployments of a fungible `bridgedUSDC` implementation on any chain. Additionally, it included access control restrictions for contract upgrades and migrations to native USDC. However, this design required a trusted third party to manage the relevant keys, introducing various layers of liability. These concerns did not meet the issuer's requirements at the time but may be revisited in the future.
+
+# Proposed Solution
+
+The solution involves a [Custom Token Bridge](https://docs.optimism.io/builders/app-developers/bridging/custom-bridge) implementation, which provides desired features such as pausability and the ability to perform migrations. The OP Stack security model secures the proposed design, similar to how `StandardBridge` operates today.
+
+### Implementation
+
+There are four main pieces:
```

Comment:
Are these all proxies?

Reply:
Only the bridged usdc contract will have a proxy and implementation, the rest are immutable.